model/Cas/DateTimeCzTrait.php

```diff
@@ -290,6 +291,12 @@ public function formatCasZacatekUdalosti(): string
         return parent::format(self::FORMAT_ZACATEK_UDALOSTI);
     }
 
+    /** Sníží časový údaj o jeden den. Upravuje objekt. */
+    public function minusDen(): static
+    {
+        return $this->sub(new \DateInterval('P1D'));
```

Comment:
Poznámka: `DateInterval` vypadá víc geekovksy, ale pro lidi je čitelnější `modify`, například `->modify('-1 day')`.

A ještě k tomu je `modify` o 15 % rychlejší (protože nemusí řešit PHP object instance).

Reply:
@BurgrLada Palec jsi dal, ale kód jsi nezměnil?